vllm/config/compilation.py

```diff
@@ -629,15 +631,38 @@ def set_splitting_ops_for_v1(self):
 
     def set_splitting_ops_for_inductor_graph_partition(self):
         assert self.use_inductor_graph_partition
-        use_inductor_graph_partition_msg = (
-            "When use_inductor_graph_partition=True, splitting_ops "
-            "are ignored and set to an empty list. Instead, "
-            "\"tags=(torch._C.Tag.cudagraph_unsafe, ),\" is "
-            "used to annotate custom ops for graph partition.")
-        if self.splitting_ops is not None and \
-            len(self.splitting_ops) > 0:
-            logger.warning_once(use_inductor_graph_partition_msg)
-        self.splitting_ops = []
+
+        if self.splitting_ops:
+            logger.debug(
+                "Using splitting_ops=%s for inductor partition rules.",
+                self.splitting_ops,
+            )
+            return
+
+        if self.splitting_ops == []:
+            logger.warning_once(
+                "Empty splitting_ops provided with "
+                "use_inductor_graph_partition; defaulting to attention ops.")
+        else:
+            logger.debug(
+                "No splitting_ops provided; defaulting to attention ops for "
+                "inductor partition rules.", )
+
+        self.splitting_ops = list(self._attention_ops)
+
+    def get_inductor_partition_ops(self) -> list[str]:
```

Comment:
This method feels almost fully redundant:
1. We already check `use_inductor_graph_partition` before we call it
2. We always initialize `splitting_ops`
3. We always only keep unique values